doc/mock_example/input.yaml

```diff
@@ -1,10 +1,17 @@
 #mock target configuration file
+
+dust_dir: /project/projectdirs/desi/software/edison/dust/v0_1/maps
+
+decals_brick_info: /global/homes/a/adamyers/brick-info-dr3.fits
```

Comment:
Let's not have the canonical example point to someone's home directory :)  Please put this into `/project/projectdirs/desi/target/catalogs/` and add a README explaining where it came from (command, code version if you have it, etc.). When we make a new DR3.1 catalog, we should also make a new brickinfo file with tagged traceability.